README.md

```diff
@@ -71,6 +71,20 @@ ClusterServiceVersions can be collected into `CatalogSource`s which will allow a
 
 Learn more about the components used by OLM by reading about the [architecture] and [philosophy].
 
+# Overall OLM Chart
+
+![OLM](./olm.png)
+
+From the above chart, we can see that there are 2 steps required for a new operator onboard:
+
+1. Develop your operator and push it into the operator repository. You can choose quay.io as an example repository.
+
+2. Use the operator in your kubernetes cluster or openshift cluster. Assume that you have OLM installed.
+
+  - Create `OperatorSource` manually to point to your operator repository. the `marketplace` operator is reposible for creating `CatalogSource`.
```

Comment:
`operator-marketplace` is a separate project, and is not required to install an operator from a catalog image -- only from Appregistry, which is being deprecated. 

It's valuable to talk about marketplace interactions somewhere, but I don't think it belongs in OLM's core architecture diagrams.

Reply:
@aravindhp @kevinrizza I saw the `operator-marketplace` is a core component when I use OLM in OCP 4.2. Will Appregistry be deprecated in OCP 4.x? Thanks.

Reply:
I agree with @njhale here. The `operator-marketplace` is a separate project: https://github.com/operator-framework/operator-marketplace

It is used to drive automatically updated content on clusters and helps to install the default catalogs. But it is not required to use OLM and add catalogs to a cluster.

There are plans around deprecating app-registry in OCP soon, which would make the marketplace an obsolete component: https://github.com/openshift/enhancements/blob/master/enhancements/olm/operator-registry.md

Given those compounding factors, I would recommend not mentioning it in OLM docs.

Reply:
Thanks for clarification. I will remove `operator-marketplace` soon.